integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -327,49 +374,37 @@ fn call_burn_gas_contract(
                 {
                     tracing::debug!(target: "test", height=tip.height, "resharding height set");
                     resharding_height.set(Some(tip.height));
-                    // Verify that delayed receipts are indeed in the queue.
-                    let epoch_manager = &client_actor.client.epoch_manager;
-                    let shard_id =
-                        epoch_manager.account_id_to_shard_id(&signer_id, &tip.epoch_id).unwrap();
-                    let shard_uid = &ShardUId::from_shard_id_and_layout(
-                        shard_id,
-                        &epoch_manager.get_shard_layout(&tip.epoch_id).unwrap(),
-                    );
-                    let congestion_info = &client_actor
-                        .client
-                        .chain
-                        .chain_store()
-                        .get_chunk_extra(&tip.last_block_hash, shard_uid)
-                        .unwrap()
-                        .congestion_info()
-                        .unwrap();
-                    assert_ne!(congestion_info.delayed_receipts_gas(), 0);
                 }
             }
             // Before resharding and one block after: call the test contract a few times per block.
             // The objective is to pile up receipts (e.g. delayed).
             if tip.height <= resharding_height.get().unwrap_or(1000) + 1 {
-                for _ in 0..CALLS_PER_BLOCK_HEIGHT {
-                    nonce.set(nonce.get() + 1);
-                    let method_name = "burn_gas_raw".to_owned();
-                    let burn_gas: u64 = GAS_PER_CALL;
-                    let args = burn_gas.to_le_bytes().to_vec();
-                    let tx = SignedTransaction::call(
-                        nonce.get(),
-                        signer_id.clone(),
-                        receiver_id.clone(),
-                        &signer,
-                        0,
-                        method_name,
-                        args,
-                        GAS_PER_CALL + 10 * TGAS,
-                        tip.last_block_hash,
-                    );
-                    let mut txs_vec = txs.take();
-                    tracing::debug!(target: "test", height=tip.height, tx_hash=?tx.get_hash(), "submitting transaction");
-                    txs_vec.push((tx.get_hash(), tip.height));
-                    txs.set(txs_vec);
-                    submit_tx(&node_datas, &"account0".parse().unwrap(), tx);
+                for _ in 0..(CALLS_PER_BLOCK_HEIGHT / signer_ids.len()
```

Comment:
Would it benefit to do round robin across signer_ids?
```
for i in 0..CALLS_PER_BLOCK_HEIGHT
  signer_id = signer_ids[i % signer_ids.len()]
  ...
```

Reply:
It should not make any difference I guess 🤔

Reply:
The current logic `0..(CALLS_PER_BLOCK_HEIGHT / signer_ids.len() + (CALLS_PER_BLOCK_HEIGHT % signer_ids.len()).min(1))` can exceeds CALLS_PER_BLOCK_HEIGHT right?

Reply:
It does, yes, trying to be fair to all accounts. Maybe it should respect more importantly  `CALLS_PER_BLOCK_HEIGHT` rather than fairness to help with making expectations clearer .. let me modify this part